Fixes #2 (#5)
psf/project-funding-wg: resources.md

# Resources

> As part of the [charter](https://wiki.python.org/psf/ProjectFundingWG) of the Project Funding Working Group, we offer the following resources for any organization or individual that wishes to secure funding to improve the Python ecosystem.

Reviewer: ```suggestion
Per the [charter](https://wiki.python.org/psf/ProjectFundingWG) of the Project Funding Working Group, we offer the following resources for any organization or individual that wishes to secure funding to improve the Python ecosystem.
```
